front/cypress/support/dnd.ts

```diff
@@ -0,0 +1,239 @@
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace Cypress {
```

Comment:
Once we merge this, it would be nice to add it to the FE updates and update the team that this exists in the Code Discussion Meeting 👍